chrome/locale/en-US/zotero/preferences.dtd

```diff
@@ -101,12 +101,14 @@
 
 <!ENTITY zotero.preferences.quickCopy.caption				"Quick Copy">
 <!ENTITY zotero.preferences.quickCopy.defaultOutputFormat	"Default Output Format:">
+<!ENTITY zotero.preferences.quickCopy.locale				"Bibliography Locale:">
```

Comment:
Sure. Don't really remember why I used "Bibliography Locale:". Probably because initially we didn't do the locale menu disabling, and I wanted to convey that the setting only affected style output. (and it matched with extensions.zotero.export.bibliographyLocale )